sale_default_terms_conditions/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+To use this module, you need to:
+
+#. Go do Sale App
+#. Create a New Sale/Quotation Order
```

Comment:
Please specify that the `sale_order_note` must be filled on the selected partner